content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
+listening on (default is `:6443`).
+
+The load balancer must have a resolvable DNS name (we do not suggest using an IP
+address directly in a cloud environment). The load balancer must have the ability to
+communicate with all control plane nodes on the apiserver port and it must allow
+incoming traffic on its listening port.
+
+After this step, you have a resolvable DNS name for your load balancer.
+
+You should add the control plane nodes to the load balancer but be aware they
+will fail the healthcheck until the kube-apiserver is running.
+
+### The first control plane node
+
+#### Prerequisites
+
+1. This node must have [kubeadm, kubelet and kubectl][installation] installed.
+1. The node must have all the certs specified in the etcd section above.
+
+[installation]: https://kubernetes.io/docs/tasks/tools/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl
+
+This guide does not cover cloud-provider installation. Therefore if you are in a
+cloud environment your services of type LoadBalancer will not work.
+
+Variables to replace in this kubeadm-config.yaml file:
+
+- `LOAD_BALANCER_DNS`
+- `LOAD_BALANCER_PORT`
+- `HOST_IP`
+- `ETCD_0_IP`
+- `ETCD_1_IP`
+- `ETCD_2_IP`
+
+```yaml
+apiVersion: kubeadm.k8s.io/v1alpha2
+kind: MasterConfiguration
+kubernetesVersion: v1.11.0
+apiServerCertSANs:
+- "LOAD_BALANCER_DNS"
+api:
+    advertiseAddress: "HOST_IP"
+    controlPlaneEndpoint: "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT"
+apiServerExtraArgs:
+    apiserver-count: "3"
+    endpoint-reconciler-type: lease
+etcd:
+    external:
+        endpoints:
+        - https://ETCD_0_IP:2379
+        - https://ETCD_1_IP:2379
+        - https://ETCD_2_IP:2379
+        caFile: /etc/kubernetes/pki/etcd/ca.crt
+        certFile: /etc/kubernetes/pki/apiserver-etcd-client.crt
+        keyFile: /etc/kubernetes/pki/apiserver-etcd-client.key
+networking:
+    # This CIDR is a calico default. Substitute or remove for your CNI provider.
+    podSubnet: "192.168.0.0/16"
+```
 
-    - `<private-ip>` with the private IPv4 of the master server.
-    - `<etcd0-ip>`, `<etcd1-ip>` and `<etcd2-ip>` with the IP addresses of your three etcd nodes
-    - `<podCIDR>` with your Pod CIDR. Please read the [CNI network section](/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the docs for more information. Some CNI providers do not require a value to be set.
-    - `<load-balancer-ip>` with the virtual IP set up in the load balancer. Please read [setting up a master load balancer](/docs/setup/independent/high-availability/#set-up-master-load-balancer) section of the docs for more information.
+Everything is now in place so you can run `kubeadm init --config kubeadm-config.yaml`
 
-    **Note:** If you are using Kubernetes 1.9+, you can replace the `apiserver-count: 3` extra argument with `endpoint-reconciler-type: lease`. For more information, see [the documentation](/docs/admin/high-availability/#endpoint-reconciler).
+### Swizzling certs
 
```

Comment:
I'd add some flavor text along the lines of:

"For those unfamiliar with the term "swizzling" in this context, it essentially refers to copying the certificates generated  in the steps above to other master hosts."

Reply:
Better not to use the term at all. Too much insider-speak. A better option (in use elsewhere in the docs, too): "Copy certificates" (don't need gerund form of verb, either). In answer to Ruben's question on Slack: "certificates" is preferable because clearer to non-native speakers/readers of English; abbreviations are not so reader-friendly in general. Yes, "certs" appears all over the docs, but we fix what we can when we encounter it.

Reply:
lol i was not even aware how many times i used certs, thanks for pointing this out.